highfidelity/hifi
# Shape type and compound shape property fixes

https://highfidelity.manuscript.com/f/cases/20735
https://highfidelity.manuscript.com/f/cases/20777
https://highfidelity.manuscript.com/f/cases/20801

Pre-merge test plan:
- Enter Interface in desktop mode into any domain with edit permissions
- Open Create and select or create any Model entity
- Verify you can change the model's Collision Shape and Compound Shape (for compound shape setting, set a URL to an OBJ or FBX file) properties as expected
- Exit and re-enter Create, return to the same model entity from above, verify the Collision Shape and Compound Shape properties are the same as before
- Select or create any Zone entity
- Verify the Shape Type and Compound Shape URL properties for Zones appear immediately under Parent Joint Index at the top
- Verify you can change the zone's Shape Type and Compound Shape URL (for use compound shape URL setting, set a URL to an OBJ or FBX file) properties as expected
- Also verify tooltips appear for Shape Type and Compound Shape URL properties as expected and start with "The model file"
- Exit and re-enter Create, return to the same zone entity from above, verify the Shape Type and Compound Shape URL properties are the same as before
